pdh_check/tests/test_e2e.py

```diff
@@ -0,0 +1,16 @@
+# (C) Datadog, Inc. 2010-2017
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+from copy import deepcopy
+
+import pytest
+from .common import INSTANCE
+from datadog_test_libs.win.pdh_mocks import initialize_pdh_tests, pdh_mocks_fixture  # noqa: F401
+
+
+@pytest.mark.e2e
```

Comment:
Add skip if not windows

Reply:
Fixed by https://github.com/DataDog/integrations-core/pull/4398